packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_controller.dart

```diff
@@ -81,6 +84,27 @@ class PropertyEditorController extends DisposableController
     super.dispose();
   }
 
+  @override
+  void filterData(Filter<EditableProperty> filter) {
+    super.filterData(filter);
+    final filtered = (_editableWidgetData.value?.properties ?? []).where(
+      (property) => property.matchesSearchToken(
+        RegExp(filter.queryFilter.query, caseSensitive: false),
+      ),
+    );
+    filteredData
+      ..clear()
+      ..addAll(filtered);
+  }
+
+  @override
+  void setActiveFilter({
+    String? query,
+    SettingFilters<EditableArgument>? settingFilters,
+  }) {
+    super.setActiveFilter(query: query);
```

Comment:
do we even need an override here since this is just calling super?

Reply:
Ah oops no!